benchmarks/Neo.VM.Benchmarks/Benchmarks.POC.cs

```diff
@@ -98,15 +98,299 @@ public void NeoIssue2723()
+            // DROP
+            // LDLOC 00
+            // DEC
+            // STLOC 00
+            // LDLOC 00
+            // JMPIF_L f4ffffff
+            // RET
+            Run("VwEAAlkNeU93AAH4B8ZFbwCddwBvACX0////QA==");
+        }
+
+        [Benchmark]
+        public void PoC_Roll()
+        {
+            // INITSLOT 0100
+            // PUSHINT16 2040
+            // STLOC 00
+            // PUSHDATA1 aaabbbbbbbbbcccccccdddddddeeeeeeefffffff
+            // LDLOC 00
+            // DEC
+            // STLOC 00
+            // LDLOC 00
+            // JMPIF_L cfffffff
+            // PUSHINT32 23000000
+            // STLOC 00
+            // PUSHINT16 2039
+            // ROLL
+            // LDLOC 00
+            // DEC
+            // STLOC 00
+            // LDLOC 00
+            // JMPIF_L f5ffffff
+            // CLEAR
+            // RET
+            Run("VwEAAfgHdwAMKGFhYWJiYmJiYmJiYmNjY2NjY2NkZGRkZGRkZWVlZWVlZWZmZmZmZmZvAJ13AG8AJc////8CwPNeAXcAAfcHUm8AnXcAbwAl9f///0lA");
+        }
+
+        [Benchmark]
+        public void PoC_XDrop()
+        {
+            // INITSLOT 0100
+            // PUSHINT16 2040
+            // STLOC 00
+            // PUSHDATA1 aaabbbbbbbbbcccccccdddddddeeeeeeefffffff
+            // LDLOC 00
+            // DEC
+            // STLOC 00
+            // LDLOC 00
+            // JMPIF_L cfffffff
+            // PUSHINT32 23000000
+            // STLOC 00
+            // PUSHINT16 2039
+            // XDROP
+            // DUP
+            // LDLOC 00
+            // DEC
+            // STLOC 00
+            // LDLOC 00
+            // JMPIF_L f4ffffff
+            // CLEAR
+            // RET
+            Run("VwEAAfgHdwAMKGFhYWJiYmJiYmJiYmNjY2NjY2NkZGRkZGRkZWVlZWVlZWZmZmZmZmZvAJ13AG8AJc////8CwPNeAXcAAfcHSEpvAJ13AG8AJfT///9JQA==");
+        }
+
+        [Benchmark]
+        public void PoC_MemCpy()
+        {
+            // INITSLOT 0100
+            // PUSHINT32 1048576
+            // NEWBUFFER
+            // PUSHINT32 1048576
+            // NEWBUFFER
+            // PUSHINT32 133333337
+            // STLOC 00
+            // OVER
+            // PUSH0
+            // PUSH2
+            // PICK
+            // PUSH0
+            // PUSHINT32 1048576
+            // MEMCPY
+            // LDLOC 00
+            // DEC
+            // STLOC 00
+            // LDLOC 00
+            // JMPIF_L eeffffff
+            // CLEAR
+            // RET
+            Run("VwEAAgAAEACIAgAAEACIAlmB8gd3AEsQEk0QAgAAEACJbwCddwBvACXu////SUA=");
+        }
+
+        [Benchmark]
+        public void PoC_Unpack()
+        {
+            // INITSLOT 0200
+            // PUSHINT16 1010
+            // NEWARRAY
+            // STLOC 01
+            // PUSHINT32 1333333337
+            // STLOC 00
+            // LDLOC 01
+            // UNPACK
+            // CLEAR
+            // LDLOC 00
+            // DEC
+            // STLOC 00
+            // LDLOC 00
+            // JMPIF_L f5ffffff
+            // RET
```

Comment:
The same comment is relevant for the rest of benchmarks. We may safely skip all opcodes like RET or opcodes that clean VM after execution, it's not related to the benchmark itself.

Reply:
> The same comment is relevant for the rest of benchmarks. We may safely skip all opcodes like RET or opcodes that clean VM after execution, it's not related to the benchmark itself.

RET is not part of the script? I converted them from script in neo's website.....

Reply:
But in these benchmarks, we are talking about execution time of seconds or milliseconds,,, should be fine with a RET here.